hokusai/test.yml

```diff
@@ -3,6 +3,5 @@ version: '2'
 services:
   frequency:
     command: ["echo", "no", "tests"]
-    extends:
-      file: build.yml
-      service: frequency
+    build:
+      context: ../
```

Comment:
otherwise build fails with:

```
jian@artsy:~/code/frequency$ hokusai test                    
ERROR: .FileNotFoundError: [Errno 2] No such file or directory: '/home/jian/code/frequency/.hokusai-tmp/build.yml'
ERROR: Command 'docker-compose -f /home/jian/code/frequency/.hokusai-tmp/tmpw7VHrC -p hokusai build' returned non-zero exit status 1
```
also agrees with [template](https://github.com/artsy/artsy-hokusai-templates/blob/master/rails-puma/hokusai/test.yml.j2).

by the way, a lot of projects have this convention:
```
jian@artsy:~/code$ grep compose-service */hokusai/test.yml
acceleration/hokusai/test.yml:{% include 'templates/docker-compose-service.yml.j2' %}
admin_metadata/hokusai/test.yml:{% include 'templates/docker-compose-service.yml.j2' %}
```
should we update template to be ^ ?

Reply:
Did this change at some point (when the variables and temp directory were introduced perhaps)?

I never saw the appeal of resorting to a separate template for just a line or 2 of YAML. Whatever's easier is fine with me for this minimal project.

Reply:
> Did this change at some point (when the variables and temp directory were introduced perhaps)?

Yes we dropped extends support as the temp directory as cross-referencing between files introduced a lot of bugs, and docker-compose eventually dropped the extends keyword.

In any case yes we should use a template fragment or just inline the context if there are no build args or other advanced build config we need to share between services.

Reply:
okay cool. leaving it as is for this project.